commit 3866e8312346839a7b6d1ddb521c4a0a8033dd5f
Author: Guy Harris <gharris@steve.local>
Date:   Thu Mar 19 09:37:03 2009 -0700

    From Romain Francoise: rename the USB-sniffing devices to "usbmonN", so
    as not to collide with the "usbN" names for USB-based networking
    devices.

---
 inet.c           |    2 +-
 pcap-linux.c     |    2 +-
 pcap-usb-linux.c |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/inet.c
+++ b/inet.c
@@ -669,7 +669,7 @@
 	    || strstr(device, "bluetooth") != NULL
 #endif
 #ifdef PCAP_SUPPORT_USB
-	    || strstr(device, "usb") != NULL
+	    || strstr(device, "usbmon") != NULL
 #endif
 	    ) {
 		*netp = *maskp = 0;
--- a/pcap-linux.c
+++ b/pcap-linux.c
@@ -322,7 +322,7 @@
 #endif
 
 #ifdef PCAP_SUPPORT_USB
-	if (strstr(device, "usb")) {
+	if (strstr(device, "usbmon")) {
 		return usb_create(device, ebuf);
 	}
 #endif
--- a/pcap-usb-linux.c
+++ b/pcap-usb-linux.c
@@ -61,7 +61,7 @@
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 
-#define USB_IFACE "usb"
+#define USB_IFACE "usbmon"
 #define USB_TEXT_DIR "/sys/kernel/debug/usbmon"
 #define USB_BUS_DIR "/proc/bus/usb"
 #define USB_LINE_LEN 4096
